### PR TITLE
Update the organization to module_manifest_org for virtwho plugin cases 

### PR DESCRIPTION
**Pull request by hkx303**
**Description**
Update the organization from `default_org` to `module_manifest_org` for virtwho plugin cases to fix failed cases. Those errors were imported from https://github.com/SatelliteQE/robottelo/pull/8993

**Test Result**
All test cases work fine locally.

### CODE REVIEW

File: robottelo/virtwho_utils.py [hunk 1]
```diff
@@ -407,22 +410,23 @@ def virtwho_package_locked():
     assert "Packages are locked" in result[1]
 
 
-def create_http_proxy(name=None, url=None, type='https'):
+def create_http_proxy(name=None, url=None, http_type='https', org_name=DEFAULT_ORG):
```
Comment (mshriver) [inline robottelo/virtwho_utils.py:413]: Only thing I would recommend here is to pass the `org` instance, and reference the field you need within this function (name, label, etc)

**Review by mshriver — APPROVED** [review]

File: robottelo/virtwho_utils.py [hunk 2]
```diff
@@ -407,22 +410,23 @@ def virtwho_package_locked():
...
     :param url: URL of the proxy including schema (https://proxy.example.com:8080)
-    :param type: https or http
+    :param http_type: https or http
+    :param org_name: name of the organization
     :return:
     """
-    default_org = entities.Organization().search(query={'search': f'name="{DEFAULT_ORG}"'})[0]
+    org = entities.Organization().search(query={'search': f'name="{org_name}"'})[0]
     http_proxy_name = name or gen_string('alpha', 15)
     http_proxy_url = url or '{}:{}'.format(
```
Comment (swadeley) [inline robottelo/virtwho_utils.py:424]: Can you try use f string here? I notice there are quite a few ".format" in the file, a job for another PR to fix them.

File: robottelo/virtwho_utils.py (hunk 1 above)
Comment (hkx303) [reply robottelo/virtwho_utils.py]: Updated, thanks

File: robottelo/virtwho_utils.py (hunk 2 above)
Comment (hkx303) [reply robottelo/virtwho_utils.py]: Good catch! Copied that from another file before, updated.

**Comment by JacobCallahan** [comment]
@swadeley can you revisit?

**Review by swadeley — APPROVED** [review]
ACK

**Pull request merged**